vllm/mocks/mock_nixl_connector.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+import uuid
+from collections import defaultdict
+from typing import Optional
+
+
+class FakeNixlWrapper:
```

Comment:
Is there no way to keep the mock under tests? I would think we'd want to avoid including it in the shipped vllm package/wheel.

Reply:
yeah I'd also prefer that. But I could find any solution that works for mocking when using ray. is there any harm in shipping them? I couldn't find any solution that would work with mocking. cc @ruisearch42 if he has any ideas.